RAD-Seq/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Galaxy - RAD-Seq training material
+
+FROM engineson/galaxy-rad-seq
```

Comment:
Do you need any training material?
Do you want to include the Galaxy RAD seq tour?